lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -633,6 +633,19 @@ radius_server:
     config_set: '<state> radius-server host <ip> key <key>'
     default_value: null
 
+radius_server_group:
+  groups:
+    config_get: "show running-config radius all"
+    config_get_token: '/^aaa group server radius (.*) $/'
```

Comment:
Please use this instead:
```
config_get_token: '/^aaa group server radius (\S+)/'
```
We like the anchors but this way we can reuse it for eXR (which doesn't have the trailing space).